uaclient/api/u/pro/security/fix/_common/plan/v1.py

```diff
@@ -700,6 +707,7 @@ def _fix_plan_cve(issue_id: str, cfg: UAConfig) -> FixPlanResult:
         exceptions.SecurityAPIError,
     ) as e:
         fix_plan = get_fix_plan(title=issue_id)
+        fix_plan.current_status = "error"
```

Comment:
I think current_status can be a default of `None` in error cases - then we don't need to add this to every possible error scenario - WDYT?

Reply:
I see why `None` would be a better option. Thanks!